src/add.test.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import addTask from './add';
+
+class LocalStorageMock {
+  constructor() {
+    this.store = {};
+  }
+
+  clear() {
+    this.store = {};
+  }
+
+  getItem(key) {
+    return this.store[key] || null;
+  }
+
+  setItem(key, value) {
+    this.store[key] = String(value);
+  }
+
+  removeItem(key) {
+    delete this.store[key];
+  }
+}
+
+document.body.innerHTML = `<input id="taskInput" value="Something">
+    <i class="far fa-calendar-plus"></i>
+    <div id="list"></div>
+    <i class="clearBtn"></i>`;
+
```

Comment:
Kindly fix this error for your add functionality test

<img width="808" alt="Screenshot 2021-07-29 at 17 06 56" src="https://user-images.githubusercontent.com/34798898/127526685-d16738d2-472e-48ae-8e8f-c7c3efc3ffd0.png">

Reply:
It is passing the tests. This error comes from this command location.reload(); if I remove this the app breaks everywhere and can not be fixed.